package.json

```
@@ -1,39 +1,90 @@
 {
-  "name": "@heroku-cli/heroku-repo",
+  "name": "@heroku-cli/plugin-heroku-repo",
   "description": "Heroku CLI plugin to manipulate the repo",
   "version": "1.0.14",
   "author": "Heroku",
   "bugs": {
     "url": "https://github.com/heroku/heroku-repo/issues"
   },
   "dependencies": {
-    "@heroku-cli/plugin-run": "3.8.10",
+    "@heroku/http-call": "^5.4.0",
+    "@heroku-cli/color": "2.0.1",
+    "@heroku-cli/command": "^11.4.0",
+    "@heroku-cli/notifications": "^1.2.5",
+    "@oclif/core": "^2.16.0",
     "bytes": "3.0.0",
-    "co": "4.6.0",
-    "heroku-cli-util": "8.0.12",
+    "debug": "^4.4.0",
     "mkdirp": "0.5.1",
     "smooth-progress": "1.1.0"
   },
   "devDependencies": {
-    "standard": "^11.0.1"
+    "@heroku-cli/schema": "^2.0.0",
+    "@oclif/plugin-help": "^5",
+    "@types/bytes": "^3.1.4",
+    "@types/chai": "^4",
+    "@types/debug": "^4.1.12",
+    "@types/mocha": "^10.0.1",
+    "@types/nock": "^11.1.0",
+    "@types/node": "^20.14.8",
+    "@types/proxyquire": "^1.3.31",
+    "@types/sinon": "^17.0.3",
+    "@typescript-eslint/eslint-plugin": "^6",
+    "@typescript-eslint/parser": "^6",
+    "chai": "^4.4.1",
+    "eslint": "^7",
+    "eslint-config-oclif": "^4",
+    "eslint-config-oclif-typescript": "^1",
+    "eslint-import-resolver-typescript": "^3",
+    "eslint-plugin-import": "^2",
+    "eslint-plugin-mocha": "^10.5.0",
+    "mocha": "^11.1.0",
+    "nock": "^14.0.0",
+    "nyc": "^17.1.0",
+    "oclif": "^4.17.13",
+    "proxyquire": "^2.1.0",
+    "sinon": "^19.0.2",
+    "stdout-stderr": "^0.1.13",
+    "strip-ansi": "^6",
+    "ts-node": "^10.9.1",
+    "tsheredoc": "^1.0.1",
+    "typescript": "4.8.4"
+  },
+  "engines": {
+    "node": ">= 20"
   },
   "homepage": "https://github.com/heroku/heroku-repo",
   "keywords": [
-    "heroku-plugin"
+    "heroku-plugin",
+    "oclif"
   ],
   "files": [
     "/index.js",
     "/commands",
     "/lib"
   ],
```

I believe this needs to be replaced with:
```suggestion
  "files": [
    "/dist"
  ],
```

---

Good catch, thank you!